packages/formatter-stylish/README.md

```diff
@@ -0,0 +1,14 @@
+# Formatter stylish (`@sonarwhal/formatter-stylish`)
+
+The `stylish` formatter prints the results in table format indicating the
+resource, line, and column:
+
+![Example output for the stylish formatter](images/stylish-output.png)
```

Comment:
Move `images/` into `src/` so that they are copied into `dist/` and get publish with the rest.

Reply:
Done, I've changed the `package.json`